### PR TITLE
pulumi-whoami: add page

### PR DESCRIPTION
**Pull request by nelsonfigueroa**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

File: pages/common/pulumi-whoami.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- Print the username of the currently logged in user:
+
+`pulumi whoami`
+
+- Print detailed information about the currently logged in user:
+
+`pulumi whoami {{-v|--verbose}}`
+
+- Print detailed information about the currently logged in user in JSON format:
```
Comment (spageektti) [inline pages/common/pulumi-whoami.md:14]:
```suggestion
- Print the username of the currently logged-in user:

`pulumi whoami`

- Print detailed information about the currently logged-in user:

`pulumi whoami {{-v|--verbose}}`

- Print detailed information about the currently logged-in user in JSON format:
```

File: pages/common/pulumi-whoami.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# pulumi whoami
+
+> Print information about the current logged-in user.
```
Comment (spageektti) [inline pages/common/pulumi-whoami.md:3]:
```suggestion
> Print information about the current logged in user.
```

**Review by spageektti — COMMENTED** [review]
hi. Thank you so much for your contributions. I have left you two suggestions, but only one should be merged.

File: pages/common/pulumi-whoami.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# pulumi whoami
+
+> Print information about the current logged in user.
```
Comment (spageektti) [inline pages/common/pulumi-whoami.md:3]:
```suggestion
> Print information about the currently logged in user.
```

File: pages/common/pulumi-whoami.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Print the username of the currently logged in user:
+
+`pulumi whoami`
+
+- Print detailed information about the currently logged in user:
+
+`pulumi whoami {{-v|--verbose}}`
+
+- Print detailed information about the currently logged in user in JSON format:
```
Comment (acuteenvy) [inline pages/common/pulumi-whoami.md:14]:
```suggestion
- Print detailed information about the currently logged in user as JSON:
```

**Review by acuteenvy — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

**Pull request merged**